### PR TITLE
Your Health Matters - Edna

### PR DESCRIPTION
**Pull request by wylie**
Cleaning up the buttons. A lot of the styling for our new flat buttons was living in multiple places within Buzz and the button styles in Edna were old. So I moved all the new styles in Buzz back into Edna.

Another part of this PR changes the styling for the Edna Codeguide markup blocks.

/cc @gjjones

### CODE REVIEW

**Comment by wylie** [comment]
Would you look at those line changed numbers! -411, boom!
screen shot 2015-07-24 at 11 31 33 am

File: less/_input-group.less [hunk 1]
```diff
@@ -120,6 +120,7 @@
 	border-top-right-radius: 0;
 	border-bottom-right-radius: 0;
 }
+// this is overriding some border radii and requiring the addition of border-radius: 3px; in Buzz
```
Comment (gjjones) [inline less/_input-group.less:123]: At some point, should this get moved out of Edna if it's a Buzz thing?

File: less/_variables.less [hunk 2]
```diff
@@ -191,7 +157,7 @@
 @input-height: @btn-height;
 @input-reset-bgcolor: transparent;
 @input-reset-bordercolor: @searchbar-reset-bordercolor;
-@input-disabled-bordercolor: @btn-disabled-bordercolor;
+@input-disabled-bordercolor: @btn-disabled-border;
```
Comment (gjjones) [inline less/_variables.less:160]: what?

**Comment by amyh** [comment]
+1

File: less/_variables.less (hunk 2 above)
Comment (wylie) [inline less/_variables.less:160]: old variable vs new variable

File: less/_input-group.less (hunk 1 above)
Comment (wylie) [inline less/_input-group.less:123]: Definitely. Doing this task showed me that a lot of cleanup is needed with the inputs.

File: less/_variables.less (hunk 2 above)
Comment (gjjones) [inline less/_variables.less:160]: Seems like a weird change to make when the others consistently end in "color". I'll get over the difference, mostly just wanted to know why.

File: less/_input-group.less (hunk 1 above)
Comment (gjjones) [inline less/_input-group.less:123]: Glad to hear we've got work to do!

**Comment by gjjones** [comment]
:+1:

**Pull request merged**